### ISSUE TITLE
Add BERT finetune bells and whistles

### ISSUE THREAD

**Issue by wesg52** [CornellDataScience/IntSys-Sentiment-Summary#31]
-Add gradient accumulation
-Add evaluation sets
